Dockerfile

```
@@ -17,6 +17,8 @@ RUN pnpm build
 # production stage
 FROM alpine:3.21
 
+ARG VERSION_TAG
```

A default value would be great to avoid a warning when building manually
```suggestion
ARG VERSION_TAG=latest
```

---

Great idea, thanks.